scripts/install/RegisterLoginController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2023 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoCe\scripts\install;
+
+use oat\oatbox\extension\InstallAction;
+use oat\tao\model\mvc\DefaultUrlService;
+
+class RegisterLoginController extends InstallAction
+{
+    public function __invoke($params)
+    {
+        $service = $this->getServiceLocator()->get(DefaultUrlService::SERVICE_ID);
+        $loginOption = $service->getOption('login');
+
+        $loginOption['fallback'] = $loginOption;
+        $loginOption['ext'] = 'taoCe';
```

Comment:
Why you've chosen taoCe extension for that login proxy and not https://github.com/oat-sa/extension-tao-premium for example?

Reply:
These changes bring the ability to change the logout redirect by an env var. I believe that it can benefit not only the premium but the product in general. That's why I decided to add it here.